R/bs_sass.R

```diff
@@ -107,10 +107,23 @@ bootstrap <- function(
     )
   }
 
+  out_file_dir <- dirname(out_file)
+
   js <- bootstrap_javascript(version, TRUE)
-  success <- file.copy(js, dirname(out_file), overwrite = TRUE)
-  if (!success) {
-    warning("Failed to copy over bootstrap's javascript into the htmlDependency() directory.")
+  success_js_file <- file.copy(js, out_file_dir, overwrite = TRUE)
+  success_js_map <- local({
+    js_map_file <- paste0(js, ".map")
+    # if `version` is `"3"`, then `js` is `"bootstrap-sass/javascripts/bootstrap[.min].js"`,
+    ## which does not have a map file
+    ## so check for map file existence
+    if (file.exists(js_map_file)) {
+      file.copy(js_map_file, out_file_dir, overwrite = TRUE)
+    } else {
+      TRUE
+    }
+  })
```

Comment:
Instead, we should remove the sourcemap link in the javascript file when updating the bootstrap files in `./tools`.

If users want access to the non-minified files, they can use `minified = FALSE`.